js/controllers/window.js

```diff
@@ -52,9 +52,11 @@ WindowController.prototype.initUI_ = function() {
 WindowController.prototype.initSidebar_ = function() {
   if (this.settings_.get('sidebaropen')) {
     $('#sidebar').css('width', this.settings_.get('sidebarwidth') + 'px');
+    $('#sidebar').css('border-right-width', '2px');
```

Comment:
good one! this (and some of the other style setting code in this method) should probably be handled in css rather than here. can you please add a comment at the start of this method:

// FIXME: move style setting to css where possible

Reply:
I could; although, some of the sidebar code is handled through the app's settings. For example, the state of the sidebar and its width.

The closing sidebar is done by setting its width to 0px, and opening it is done by setting it back to `n`px, where `n` is the previous width that was stored in the app settings.

Due to the nature of the way the sidebar opens, adding a class (say, `.sidebar-open`) might not be the correct way to go about it, since its width is variable.

Regardless, I'll add the comment.

Reply:
yeah some of it won't be able to be moved, which is why i said 'where possible'.